cypress/support/commands.js

```diff
@@ -257,8 +300,12 @@ Cypress.Commands.add('closeRHS', () => {
 
 Cypress.Commands.add('createNewTeam', (teamName, teamURL) => {
     cy.visit('/create_team');
-    cy.get('#teamNameInput').type(teamName).type('{enter}');
```

Comment:
@thekiiingbob Nit: Can you revert the changes from single line to multi line commands?. They are fine as they were and it would be nice to have `git blame` point to the commit which made functional changes.

Reply:
@sudheerDev Should be all reverted now!

Reply:
@thekiiingbob Thanks for reverting just one more thing. If you can merge master and update the PR i can merge this.  Mention me or saturninoabril when you update so we don't miss merging this.